File: 은지/LV2/가장 큰 수.js
function solution(numbers) {
  var answer = '';
  var num = []
  for (let x of numbers) {
    num.push(String(x).repeat(3))
  }
  num.sort().reverse()
  for (let x of num) {
    answer += x.slice(0, parseInt(x.length/3))
  }
  if (answer[0] === '0') {
    answer = '0'
  }
  return answer;
}

numbers = [0, 0, 0]
console.log(solution(numbers))

File: 은지/LV2/가장 큰 수.py
def solution(numbers):
    answer = ''
    num = [str(x)*3 for x in numbers]
    num.sort(reverse=True)
    for x in num:
        answer += x[:len(x)//3]
    answer = str(int(answer))
    return answer

numbers = [0, 0, 0]
print(solution(numbers))